pages/message-board.html

```diff
@@ -137,16 +137,25 @@ <h1 id="title-header">The Regis Altare Birthday Project 2023</h1>
     </nav>
 
       <section id="message-board">
-        <a href="https://www.kudoboard.com/boards/qQ8344eM" target="_blank"
-          >Go to message wall</a
-        >
-        <video autoplay muted loop controls>
-          <source
-            src="https://res.cloudinary.com/dazcxdgiy/video/upload/v1669320701/kudoboard_qbu1gi.mp4"
-            type="video/mp4"
-          />
-          Your browser does not support the video tag.
-        </video>
+        
+        <!-- Link out to kudo board -->
+        <p>This preview is best viewed on desktop.
```

Comment:
"This preview is best viewed on desktop." I assume is an old message from when we tried the giga JPEG. I think we can delete this sentence now?

Reply:
I added it because I considered that even with the video it was still best viewed on Desktop, because I don't expect a tiny video would be very legible to those on their phone.

Reply:
good point, I'm good with keeping it then